WooCommerce/Classes/ViewRelated/Orders/Order Creation/CustomerSection/CustomerSelectorViewModel.swift

```diff
@@ -5,12 +5,16 @@ final class CustomerSelectorViewModel {
     private let stores: StoresManager
     private let siteID: Int64
 
+    let configuration: CustomerSelectorViewController.Configuration
```

Comment:
The `configuration` doesn't seem to be used in the view model - should we move it to the view controller for simplicity?

Reply:
During the meetup I got a suggestion from @selanthiraiyan to put this into the viewmodel, to make it testable (which doesn't really show now since there is no unit tests added yet).

I might have misunderstood the suggestion due to the meetup daze, though, so looking forward to Sharma's opinion on this.

Reply:
Thanks for adding a `configuration`. If we don't have any logic in view model (which can be unit tested) based on the injected configuration we can move this to view controller. 👍

Reply:
Thanks Sharma, this is done in https://github.com/woocommerce/woocommerce-ios/pull/12100/commits/fc448cdf7d111b2e623a25623743c2ba70082401